testsuite/scripts/src/test/java/org/wildfly/scripts/test/DomainScriptTestCase.java

```diff
@@ -15,22 +19,34 @@
 import org.jboss.as.test.shared.TestSuiteEnvironment;
 import org.jboss.dmr.ModelNode;
 import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 import org.wildfly.common.test.ServerHelper;
 
 /**
  * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
  */
+@RunWith(Parameterized.class)
 public class DomainScriptTestCase extends ScriptTestCase {
 
     private static final Function<ModelControllerClient, Boolean> HOST_CONTROLLER_CHECK = ServerHelper::isDomainRunning;
 
+    @Parameterized.Parameter
+    public Map<String, String> env;
     public DomainScriptTestCase() {
         super("domain");
     }
 
+    @Parameterized.Parameters
+    public static Collection<Object> data() {
+        return List.of(Map.of(), Map.of("SECMGR", "true"));
+    }
+
     @Override
     void testScript(final ScriptProcess script) throws InterruptedException, TimeoutException, IOException {
-        script.start(HOST_CONTROLLER_CHECK, ServerHelper.DEFAULT_SERVER_JAVA_OPTS);
+        String[] modifiedJavaOpts = Arrays.copyOf(ServerHelper.DEFAULT_SERVER_JAVA_OPTS, ServerHelper.DEFAULT_SERVER_JAVA_OPTS.length + 1);
+        modifiedJavaOpts[modifiedJavaOpts.length - 1] = "-secmgr";
```

Comment:
Sorry, I was clear with my wording here. We shouldn't be adding the `-secmgr` parameter to the server arguments if we've already got it set in the environment variable. Essentially we could just do this:
```java
script.start(HOST_CONTROLLER_CHECK, env, ServerHelper.DEFAULT_SERVER_JAVA_OPTS);
```